standard/page.lua

```diff
@@ -0,0 +1,61 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Page
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local String = require('Module:StringUtils')
+
+local page = {}
+
+function page.exists(page)
+	local existingPage = mw.title.new(page)
+
+	-- In some cases we might have gotten an external link,
+	-- which will mean `existingPage` will equal nil
+	if existingPage == nil then
+		return false
+	end
+
+	return existingPage.exists
+end
+
+function page.makeInternalLink(link, display)
+	if String.isEmpty(link) then
+		return nil
+	elseif String.isEmpty(display) then
+		display = link
+	end
+
+	return '[[' .. link .. '|' .. display .. ']]'
+end
```

Comment:
could also switch the roles of display and link around a bit:
* make display the needed value and customLink optional
```suggestion
function page.makeInternalLink(display, customLink)
	if String.isEmpty(display) then
		return nil
	elseif String.isEmpty(customLink) then
		customLink= display
	end

	return '[[' .. customLink .. '|' .. display .. ']]'
end
```
tell me which you prefer, will adjust it here and in the other functions if wanted^^

Reply:
Yes I like `display, customLink`